index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile Version</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>   
+    <div class="wrapper">
```

Comment:
Let's try to separate the sections, we have just two sessions in this project for now.

Let's use the `header` tag to wrap the `nav` tag.
Then use `display: flex` on the header tag, and you can also add `justify-content: space-between;`.

Now, do the same for the second session using flexbox.

Example:
<header>....
<section>...


@leonardoeng13 I hope this helps.

Reply:
I think now it's working ... I found the readme image error ... case sensitive in the readme I wrote png and the file are PNG

Reply:
I know you guys are pushing me to be better, but this 1 first week was hard for me to got myself on the tracks. Could you please approve my code?
I promise I'll review my code this weekend to improve myself, but now I'm running out of time.